lib/common/crmcommon_private.h

```diff
@@ -103,6 +103,9 @@ void pcmk__apply_creation_acl(xmlNode *xml, bool check_top);
 G_GNUC_INTERNAL
 void pcmk__mark_xml_attr_dirty(xmlAttr *a);
 
+G_GNUC_INTERNAL
+bool pcmk__xa_filterable(const char *name);
+
```

Comment:
Should I have been marking all the pcmk__ functions I've been creating with G_GNUC_INTERNAL?

Reply:
Only functions that are not used outside the library they're in -- hence why this header is in lib/common rather than include